ckanext/dcatapchharvest/profiles.py

```diff
@@ -846,15 +861,11 @@ def graph_from_dataset(self, dataset_dict, dataset_ref):  # noqa C901
         # Languages
         languages = dataset_dict.get("language", [])
         for lang in languages:
-            if "http://publications.europa.eu/resource/authority" in lang:
-                # Already a valid EU language URI
-                g.add((dataset_ref, DCT.language, URIRef(lang)))
+            uri = language_uri_map.get(lang, None)
+            if uri:
+                g.add((dataset_ref, DCT.language, URIRef(uri)))
             else:
-                uri = language_uri_map.get(lang, None)
-                if uri:
-                    g.add((dataset_ref, DCT.language, URIRef(uri)))
-                else:
-                    log.debug(f"Language '{lang}' not found in language_uri_map")
+                log.debug(f"Language '{lang}' not found in language_uri_map")
```

Comment:
The values in the language list of the resource dict from CKAN can only be the ones in the language field's choices as defined in the dataset schema: 'de', 'en', 'fr' or 'it'.